oqpy/quantum_types.py

```
@@ -57,7 +57,7 @@ def make_declaration_statement(self, program: Program) -> ast.Statement:
         """Make an ast statement that declares the OQpy variable."""
         decl = ast.QubitDeclaration(
             ast.Identifier(self.name),
-            size=ast.IntegerLiteral(self.size) if self.size else self.size,
+            size=ast.IntegerLiteral(self.size) if self.size else None,
```

So this is only different if `size=0`, right (assuming type hint is respected)? Would we rather just throw an error on `size=0`?